content/docs/reference/addon-manifest.md

```diff
@@ -515,7 +528,7 @@ Keep in mind, few addons will be enabled by default. If you want your addon to b
 | - | - |
 | Type | `Array` |
 
-An array of additional information (e.g. warnings, notices) about the addon. Each item of the array is an object consisting of `type` (string) -either `warning` or `notice` - `text` (string) - the text to be displayed - and `id` (string) - the id of the information.
+An array of additional information (e.g. warnings, notices) about the addon. Each item of the array is an object consisting of `type` (string) -either `warning`. `notice` or `info` - `text` (string) - the text to be displayed - and `id` (string) - the id of the information.

```

Comment:
This should mention that `type` is optional and defaults to info.

Reply:
Now it is correct?

> An array of additional information (e.g. warnings, notices) about the addon. Each item of the array is an object consisting of `id` (string) - the id of the information, `text` (string) - the text to be displayed and optional `type` (string) - either `warning`, `notice` or `info` (default type).

Reply:
Yes, that's correct.